rbd.c

```diff
@@ -115,11 +115,12 @@ static darray(char *) blacklist_caches;
 #ifdef LIBRADOS_SUPPORTS_SERVICES
 
 #ifdef RBD_LOCK_ACQUIRE_SUPPORT
+/* rdev->priv_lock is held_*/
 static int tcmu_rbd_service_status_update(struct tcmu_device *dev,
 					  bool has_lock)
 {
-	struct tcmu_rbd_state *state = tcmur_dev_get_private(dev);
 	struct tcmur_device *rdev = tcmu_dev_get_private(dev);
+	struct tcmu_rbd_state *state = tcmur_dev_get_private(dev);
```

Comment:
This is a purely cosmetic change, right?

Reply:
This maybe introduced by my previous changes and there should be some other changes in this func but removed again, this could be removed and makes no sense.